### DISCUSSION TITLE
docker appears as inactive when started after updating

### DISCUSSION THREAD

**Discussion by fernandeusto** [louislam/dockge#389] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

After updating a docker from another platform such as portainer or ssh, in dockge appears as inactive when it has already been restarted again whit new image.
![imagen](https://github.com/louislam/dockge/assets/87156183/e7fcd409-007e-4410-9fea-66eb30f69f3e)
When in Portainer is running
![imagen](https://github.com/louislam/dockge/assets/87156183/64dbaca0-a1cc-454c-af78-40e1a2827b1d)

### 👟 Reproduction steps

Start dockge and show the docker screen, in parallel from another window, update the container, in dockge you can see how the container stops in your terminal, but while is updating and restart, dockge no longer updates the terminal.
Even after restarting dockge it still appears as inactive.
It is necessary to press the start button in the dockge and then set it to running

### 👀 Expected behavior

auto refresh docker status or with button

### 😓 Actual Behavior

![imagen](https://github.com/louislam/dockge/assets/87156183/e7fcd409-007e-4410-9fea-66eb30f69f3e)

### Dockge Version

1.4.1

### 💻 Operating System and Arch

windows 11 - Nas Synology

### 🌐 Browser

Firefox

**Comment by louislam** [comment]
> After updating a docker from another platform such as portainer

I think you can manage a stack by either Dockge or Portainer only, as Portainer cannot read the /opt/stacks directory.
